kitsune/wiki/views.py

```diff
@@ -125,6 +125,32 @@ def document(request, document_slug, template=None, document=None):
             # and OK to fall back to parent (parent is approved).
             fallback_reason = 'no_translation'
 
+    # If any local has fallback locals and the docmunet is missing localization in that local
+    # So show the fallback local's document. There can be highest two fallback local for any local
+    # Fallback locals can be add in kitsune/wiki/config.py
+    if fallback_reason == 'no_translation' and (request.LANGUAGE_CODE in FALLBACK_LOCALS.keys()):
```

Comment:
This can be more simply written as `if fallback_reason == 'no_translation' and request.LANGUAGE_CODE in FALLBACK_LOCALS:`. The `in` operator when applied with a string and a dictionary returns true if any of the keys in that dictionary equal the string.